fixtures/common.py

```diff
@@ -4,15 +4,37 @@
 from selenium.webdriver.chrome.options import Options
 from selenium import webdriver
 
-
 @pytest.fixture(scope='session')
 def driver():
-    chrome_options = Options()
-    if settings.HEADLESS:
-        chrome_options.add_argument('--headless')
-        chrome_options.add_argument('--disable-gpu')
-    chrome_options.add_argument('--window-size=512,512')
-    driver = webdriver.Chrome(chrome_options=chrome_options)
+    if settings.SAUCELABS_USERNAME:
+        if settings.BROWSER == 'ie11':
+            desired_cap = {
+                'platform': "Windows 10",
+                'browserName': "internet explorer",
+                'version': "11.103",
+                }
+        elif settings.BROWSER == 'ie10':
+            desired_cap = {
+                'platform': "Windows 7",
+                'browserName': "internet explorer",
+                'version': "10.0",
+                }
+        else:
+            desired_cap = {
+                    'platform': "Mac OS X 10.9",
+                    'browserName': "chrome",
+                    'version': "31",
+                    }
+        driver = webdriver.Remote(
+           command_executor=f'http://{settings.SAUCELABS_USERNAME}:{settings.SAUCELABS_ACCESS_KEY}@ondemand.saucelabs.com:80/wd/hub',
```

Comment:
Shouldn't it be `https`?

Reply:
Unfortunately no… Using https just hangs the test, and the documentation has plain http links too.